dydx-testnet-4/pregenesis.json

```diff
@@ -0,0 +1,2020 @@
+{
+  "genesis_time": "2023-10-11T04:19:58.475018Z",
```

Comment:
I think this is just a dummy pregenesis right?

otherwise, we should update this to be 10/17 17:00 UTC

Reply:
Yes